server/backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
+
+RUN addgroup -S nonroot \
+    && adduser -S nonroot -G nonroot
```

Comment:
Hi Conrad, not sure why but I just checked out and tried to run (docker-compose up) and saw this error:
`executor failed running [/bin/sh -c addgroup -S nonroot     && adduser -S nonroot -G nonroot]: exit code: 1`

Reply:
Hello Conrad, I commented out the two docker commands for "USER nonroot".
It seems to be running fine for me now but I don't know why the USER is needed.
Please try on your end to see if it still works for you.
Thank you.

Reply:
It works fine without them, but the checks don't pass on Github because SonarCloud throws up a security error. I'll see if I can figure out how to disable that error.

Reply:
Its considered a security best practice to avoid Dockerfiles using root user. If the dockerfile is only used locally this doesn't matter in my opinion and we could just suppress the SonarCloud warning. But for dockerfiles used to build images used on AWS, I'd push for running as a different user.